### ISSUE TITLE
Bug - Ctrl + Space not working for me

### ISSUE THREAD

**Issue by JasonSmiley** [rlogiacco/Natural#21]
Type= Error

Message=Conflicting handlers for org.eclipse.ui.edit.text.contentAssist.proposals: {org.eclipse.xtext.ui.editor.handler.ContentAssistHandler@245a9e5b} vs {ActionHandler(org.eclipse.ui.texteditor.ContentAssistAction@5baecab9)}

Exception Stack Trace=An exception stack trace is not available.

Session Data=eclipse.buildId=M20120914-1800
java.version=1.7.0_10
java.vendor=Oracle Corporation
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.java.product -keyring /Users/jason/.eclipse_keyring -showlocation
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -product org.eclipse.epp.package.java.product -keyring /Users/jason/.eclipse_keyring -showlocation

Not sure if this is a bug in the code or if it is my environment so I just wanted to open an issue. Let me know if this is just me or something everyone else is getting.

Basically, I go to a step feature file (with an implemented definition) and I hit CTRL + Space to go to the step definition when I get this error.

Please note I am using a Mac (not a PC)
